.github/pr-custom-review.yml

```
@@ -20,15 +20,16 @@ rules:
     condition:
       include: .*
       # excluding files from 'Runtime files' and 'CI files' rules
-      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*
+      exclude: ^runtime/(kusama|polkadot)/src/[^/]+\.rs$|^\.gitlab-ci\.yml|^(?!.*\.dic$)scripts/ci/.*|^\.github/.*
     min_approvals: 2
     teams:
       - core-devs
 
   - name: CI files
     check_type: changed_files
     condition:
-      include: ^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*
+      # dictionary files are excluded
+      include: ^\.gitlab-ci\.yml|^(?!.*\.dic$)scripts/ci/.*|^\.github/.*
```

```suggestion
      include: ^\.gitlab-ci\.yml|^\.github/.*
      exclude: ^scripts/ci/.*\.(dic|toml)$
```

AFAIU it was requested to remove `lingua.dic` from CI scope, as well I think `spellcheck.toml` also is not relevant to us
IMO it's better to put them into exclusions list explicitly.

---

Ups, added `^scripts/ci/.*` back

---

I agree, but you removed the whole `^scripts/ci/.*` expression from the include list and didn't add a necessary negation counterpart to the `Core developers`'s `condition:` list. I've specially used negative lookahead to circumvent such complexity here, otherwise, files that are matched with `^scripts/ci/.*\.(dic|toml)$` expression won't be parented by the `Core developers` group.

---

> Ups, added `^scripts/ci/.*` back

Yeah, and I wrote my first answer without looking at this comment, sorry :). Anyway, adding an additional `exclude:` for `CI files` here should be also reflected within `Core developers` conditions too. I circumvent such logic complications with complicating the regex itself, it's not the perfect solution, the perfect solution is to make implement "autoparenting" of dangling files by some default group within `paritytech/pr-custom-review` action itself.

Do you agree?

---

BTW, I've pushed the commit with the additional exclusion of `spellcheck.toml` file within my negative lookahead solution.

---

@joao-paulo-parity Sorry, could you please clarify if there is "default parenting rule for all files that are not matched by other rules" functionality available in the action?

---

> Sorry, could you please clarify if there is "default parenting rule for all files that are not matched by other rules" functionality available in the action?

There wasn't when I stopped working in the code (July of last year)

---

Got it, thanks.